config/arch/c6x.in

```diff
@@ -0,0 +1,37 @@
+# c6x specific configuration file
+
+## no-package
+## select ARCH_SUPPORTS_32
+## select ARCH_DEFAULT_32
+## select ARCH_SUPPORTS_EITHER_ENDIAN
+## select ARCH_DEFAULT_LE
+## select ARCH_SUPPORTS_FLAT_FORMAT
+## select ARCH_SUPPORTS_WITH_CPU
+## select TARGET_SKIP_CONFIG_SUB
+## select ARCH_REQUIRES_MULTILIB
```

Comment:
Does that really needed? This basically force selects `MULTILIB` but who stops you from selecting `MULTILIB` in the "sample" of your choice? As well as keeps an option to build a non-multilib toolchain for those how don't need libs for all kinds of processors of the same architecture.

Reply:
I think it is needed because the "-march=", "-mlittle-endian", and "-mlittle-endian" options seem to be coupled with having the four lib versions. This is where my ignorance of conventions shows.  :)   Anyways, I'm happy to remove it if that makes more sense.